addons/binding/org.openhab.binding.mihome/ESH-INF/thing/sensor_smoke.xml

```diff
@@ -9,6 +9,7 @@
 		</supported-bridge-type-refs>
 		<label>Xiaomi Smoke Sensor</label>
 		<channels>
+            <channel id="density" typeId="density"></channel>
 			<channel id="alarm" typeId="smoke_alarm"></channel>
```

Comment:
Formatting looks funny, please rember that the XML files should be formatted using tabs.

Reply:
✅